src/Exporter.php

```diff
@@ -684,6 +688,42 @@ protected function initAttributeValues()
         return $this;
     }
 
+    protected function getRequiredVariationValues():array
+    {
+        $variationValues = [];
+
+        if ($this->registry->get('categories')) {
```

Comment:
@darius-adeoweb the click-test failed because when getting those values you always get a class which extends from `\Findologic\Plentymarkets\Parser\ParserInterface`. Call `->getResults()` and check if they contain an empty array as well. E.g.
```php
$categories = $this->registry->get('categories'); 
if ($categories && !empty($categories->getResults()))
```